src/anonymous-credentials/schema.ts

```diff
@@ -1210,7 +1221,7 @@ export function getTransformedMinMax(name: string, valTyp: ValueTypes, min: numb
       transformedMax = Encoder.decimalNumberToPositiveInt(valTyp.minimum, valTyp.decimalPlaces)(max);
       break;
     default:
-      throw new Error(`${name} should be of numeric type as per schema but was ${valTyp}`);
+      throw new Error(`${name} should be of numeric type as per schema but was ${JSON.stringify(valTyp, null, 2)}`);
```

Comment:
@lovesh the reasoning behind this change is it wasnt so obvious to debug the error as itd say `but was [Object object]` in some cases